torch/nn/quantized/modules/conv.py

```diff
@@ -195,7 +195,7 @@ def get_qconv(cls, mod, activation_post_process, weight_post_process=None):
         if weight_post_process is None:
             weight_post_process = mod.qconfig.weight()
         weight_post_process(mod.weight)
-        act_scale, act_zp = activation_post_process.calculate_qparams()
```

Comment:
Why this line is removed?

Reply:
Also, if you leave it as is, what happens? Also do you not need from_float in dynamic conv module?